chemprop/cli/train.py

```diff
@@ -347,6 +347,11 @@ def add_train_args(parser: ArgumentParser) -> ArgumentParser:
         action=LookupAction(MetricRegistry),
         help="Specify the evaluation metrics. If unspecified, chemprop will use the following metrics for given dataset types: regression -> ``rmse``, classification -> ``roc``, multiclass -> ``ce`` ('cross entropy'), spectral -> ``sid``. If multiple metrics are provided, the 0-th one will be used for early stopping and checkpointing.",
     )
+    train_args.add_argument(
+        "--tracking-metric",
+        default="val_loss",
```

Comment:
We can't use the lookup action because "val_loss" isn't in the MetricRegistry. Instead we handle the logic for changing this to an actual metric later.

Reply:
fair point